tstest/integration/integration_test.go

```diff
@@ -528,22 +529,31 @@ func (n *testNode) AwaitListening(t testing.TB) {
 	}
 }
 
-func (n *testNode) AwaitIP(t testing.TB) (ips string) {
+func (n *testNode) AwaitIPSlice(t testing.TB) (ipslice []string) {
 	t.Helper()
+	var ips string
 	if err := tstest.WaitFor(20*time.Second, func() error {
 		out, err := n.Tailscale("ip").Output()
 		if err != nil {
 			return err
 		}
 		ips = string(out)
+		ipslice = strings.Fields(ips)
 		return nil
 	}); err != nil {
 		t.Fatalf("awaiting an IP address: %v", err)
 	}
-	if ips == "" {
+	if ips == "" || len(ipslice) == 0 {
 		t.Fatalf("returned IP address was blank")
 	}
-	return ips
+	return ipslice
+}
+
+// AwaitIP returns the IPv4 address of n
+func (n *testNode) AwaitIP(t testing.TB) (ips string) {
```

Comment:
It's not "ips" plural anymore. Just ip.

Would netaddr.IP be a better type at this point?